tests/test_diffusers.py

```diff
@@ -111,6 +113,7 @@
     INPAINT_THROUGHPUT_BASELINE_BF16 = 4.584
     INPAINT_XL_THROUGHPUT_BASELINE_BF16 = 1.151
     DETERMINISTIC_IMAGE_GENERATION_THROUGHPUT = 0.946
+    THROUGHPUT_UNCONDITIONAL_IMAGE_BASELINE_BF16 = 7.671212047338486
```

Comment:
I cannot reproduce this baseline on my Gaudi2 instance, the test fails

Reply:
I am able to run it alright: 
```
tests/test_diffusers.py::GaudiDDPMPipelineTester::test_no_throughput_regression_bf16 diffusion_pytorch_model.safetensors not found
Loading pipeline components...: 100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 2/2 [00:00<00:00, 10.51it/s]
[INFO|pipeline_utils.py:130] 2024-07-22 17:14:15,033 >> Enabled HPU graphs.
100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1000/1000 [01:40<00:00,  9.91it/s]
PASSED
```
What is the throughput you see? We might possibly need to lower a bit?

Reply:
@regisss What is your instance? I can also pass the same test.

Reply:
It's a Gaudi2 instance from IDC. I'm going to check again :+1: